operations/configuration_management/NCN_Node_Personalization.md

```diff
@@ -29,62 +66,151 @@ The following procedure describes how to correctly edit the `sat bootprep` files
+          "name": "analytics-integration-1.2.22",
+          "playbook": "site.yml"
         }
         ```
 
-    1. (`ncn-m#`) Edit the `management-bootprep-node-personalization.yaml` file to replace the CPE and Analytics layers with the `cloneUrl`, `commit` and `playbook` values already in use on the NCNs for CPE and Analytics.
-       This must be done because the new versions of CPE and Analytics have not yet been installed at this time in the upgrade procedure.
+    1. In the output of the previous step, note the values for `cloneUrl`, `commit`, and `playbook` for both layers.
 
-       In order to accurately represent the exact configuration already in use for CPE and Analytics, use the `git` key in the `bootprep` layer definition. Ensure that the values of `commit` and `playbook`
-       match their equivalents from the output of `cray cfs components describe`, and ensure that `url` matches the `cloneUrl` shown in that output. The example below shows what the CPE and Analytics
-       layers should look like.
+1. (`ncn-m#`) Update the `management-bootprep-node-personalization.yaml` file with the CPE and Analytics values identified in the previous step.
 
-       ```yaml
-       - name: cpe-pe_deploy-integration-{{cpe.version}}
-         playbook: pe_deploy.yml
-         git:
-           url: https://api-gw-service-nmn.local/vcs/cray/cpe-config-management.git
-           commit: 22056808ccf5e7994e75bb246df0abe550a1fe0f
-       - name: analytics-site-integration-{{analytics.version}}
-         playbook: site.yml
-         git:
-           url: https://api-gw-service-nmn.local/vcs/cray/analytics-config-management.git
-           commit: dbe24d4521155a683c52a14b39991aa3f410954e
-       ```
+    In order to accurately represent the exact configuration already in use for CPE and Analytics, use the `git` key in the `bootprep` layer definition.
+    For each of these two layers:
 
-1. (`ncn-m#`) Run `sat bootprep` against the `management-bootprep-node-personalization.yaml` file to create the CFS configuration that will be used for node personalization on management NCNs.
+    - Ensure that `commit` and `playbook` match the values identified in the previous step.
+    - Ensure that `url` matches the `cloneUrl` value identified in the previous step.
 
-    ```bash
-    sat bootprep run management-bootprep-node-personalization.yaml
+    The example below shows what the CPE and Analytics layers should look like, based on the earlier example output.
+
+    ```yaml
+    - name: cpe-pe_deploy-integration-{{cpe.version}}
+      playbook: pe_deploy.yml
+      git:
+        url: https://api-gw-service-nmn.local/vcs/cray/cpe-config-management.git
+        commit: c38c16bc9a645a9339a8303865e696b089b15e17
+    - name: analytics-site-integration-{{analytics.version}}
+      playbook: site.yml
+      git:
+        url: https://api-gw-service-nmn.local/vcs/cray/analytics-config-management.git
+        commit: c9c0b2cc69998830a47d7c989b07f550814af095
```

Comment:
Do we want to remove the `{{cpe.version}}` and `{{analytics.version}}` above?   If we leave this as is, the name will be the papaya version but the commit id will be the raspberry version.

Reply:
Good call.